src/App.js

```diff
@@ -30,7 +28,7 @@ class App extends Component {
 
     render () {
         return (
-            <Main>
+            <Main style={ { marginLeft: 0, padding: 0 } }>
```

Comment:
In the main conversation thread you mentioned this:
> I'm happy to move the main tag to the landingpage in a followup.

@jkozol was helping me with this earlier this week. We realized that when `<Main>` is not included here and when it's included in LandingPage.js, the tests fail. So he raised a question about the `<Main>` component in a google chat (cc'ing us), about what it's purpose is. 

My workaround to keep the tests working were to keep `<Main>` here in App.js, but apply the same styles that insights apps use (as [shown here](https://github.com/RedHatInsights/insights-rbac-ui/blob/d829ae500e9ee477421899df674cea234fddaf52/src/App.js#L53)) to remove the extra padding. But the html rendered by `<Main>`—i.e. `<section className="pf-l-page__main-section pf-c-page__main-section">`—is still needed on LandingPage.js, so I include just that html instead of `<Main>` in that file.

I didn't fully understand the answer that @jkozol got back when he raised the question, so I would defer to you and him on what we ultimately want to do. But if the goal is to be able to use `<Main>` in LandingPage.js instead of here, then would we still want to move the css out to a separate file? I think it makes sense to move it if this is our long-term solution, but probably easier to rip this out if it's all in one chunk like it is now.

Reply:
Alright, let's keep this as is then, and figure it out later.